dockerfiles/notebook/Dockerfile

```
@@ -2,10 +2,17 @@ FROM jupyter/datascience-notebook:python-3.10.6
 #FROM jupyter/datascience-notebook:d53a302fbcd0
 USER root
 
-# mambaでR依存パッケージをインストール
-RUN mamba install --quiet --yes r-themis==0.1.4 r-rpostgresql==0.6_2 \
-    && find / -type f -perm /u+s -ignore_readdir_race -exec chmod u-s {} \; \
-    && find / -type f -perm /g+s -ignore_readdir_race -exec chmod g-s {} \;
+# 最新のpostgresqlへの対応を行っている (参考: https://www.postgresql.org/download/linux/ubuntu/ )
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends lsb-release gnupg \
+    && sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends libpq-dev \
+    && apt-get remove -y lsb-release gnupg \
+    && apt-get autoremove -y \
+    && rm -rf /var/lib/apt/lists/*
```

```suggestion
    && rm -rf /var/lib/apt/lists/* \
    && find / -type f -perm /u+s -ignore_readdir_race -exec chmod u-s {} \; \
    && find / -type f -perm /g+s -ignore_readdir_race -exec chmod g-s {} \;
```

以下に関しては https://github.com/The-Japan-DataScientist-Society/100knocks-preprocess/pull/145 とは無関係の差分なので ( https://github.com/The-Japan-DataScientist-Society/100knocks-preprocess/pull/162 )、上記のようにして残していただけるとありがたいです。
https://github.com/The-Japan-DataScientist-Society/100knocks-preprocess/blob/5f973c83a786dcb3b376ee457276afa59cc1dcb6/dockerfiles/notebook/Dockerfile#L7-L8

---

find 系の記述を追加しました！